src/idpf.rs

```diff
@@ -690,10 +690,19 @@ where
     VL: Encode,
 {
     fn encode(&self, bytes: &mut Vec<u8>) -> Result<(), CodecError> {
-        // Control bits need to be written within each byte in LSB-to-MSB order, and assigned into
-        // bytes in big-endian order. Thus, the first four levels will have their control bits
-        // encoded in the last byte, and the last levels will have their control bits encoded in the
-        // first byte.
+        // draft-irtf-cfrg-vdaf-13, Section 8.2.6.1:
+        //
+        // struct {
+        //     opaque packed_control_bits[packed_len];
+        //     opaque seed[poplar1.idpf.KEY_SIZE*B];
+        //     Poplar1FieldInner payload_inner[Fi*poplar1.idpf.VALUE_LEN*(B-1)];
+        //     Poplar1FieldLeaf payload_leaf[Fl*poplar1.idpf.VALUE_LEN];
+        // } Poplar1PublicShare;
+        //
+        // Control bits. These need to be written within each byte in LSB-to-MSB order, and
+        // assigned into bytes in big-endian order. Thus, the first four levels will have their
+        // control bits encoded in the last byte, and the last levels will have their control bits
+        // encoded in the first byte.
```

Comment:
This comment, and the control bit packing and unpacking, need to be updated as well. We now put the bytes in the opposite order, with the first levels in the first byte.

Reply:
I'm confused ... is it just the comment that's out of date, or do you think the bit packing/unpacking code is incorrect? FWIW this is code is currently passing the test vectors on my development branch. (I'm working on a PR for updating the agg param encoding.)

Reply:
Ah nevermind, just the comment is wrong, and it's been out of sync with the implementation since #462.

Reply:
Removed the comment!